test/composertest.py

```diff
@@ -132,4 +132,8 @@ def main():
     runner = unittest.TextTestRunner(verbosity=2, failfast=args.sit)
     result = runner.run(tests)
 
+    if not result.testsRun:
+        print("No tests executed")
+        sys.exit(1)
```

Comment:
Running 0 tests is not an error.

Reply:
I agree that this is useful, but we really want to check that the number of tests we expected to run equals the number of tests that were run. Something like this:

```python
if tests.countTestCases() != result.testsRun:
     print("Error: unexpected number of tests were run", file=sys.stderr)
     sys.exit(1)
```